lib/signaling/v3/tracksubscriptionssignaling.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const MediaSignaling = require('../v2/mediasignaling');
+
+class TrackSubscriptionsSignaling extends MediaSignaling {
+  /**
+   * Construct a {@link TrackSubscriptionsSignaling}.
+   */
+  constructor(getReceiver, options) {
+    super(getReceiver, 'track_subscriptions', options);
+
+    Object.defineProperties(this, {
+      _currentRevision: {
+        value: 0,
```

Comment:
Can the initial value be unset? Using zero is ambiguous since the schema specifies the type as `uint32` and doesn't prevent the server from selecting `0` as the first revision.